src/components/widgets/gcode-preview/GcodePreview.vue

```diff
@@ -271,6 +297,9 @@ export default class GcodePreview extends Mixins(StateMixin) {
       this.flipY ? -(y.max - y.min) : 0
     ]
 
+    if (this.isDelta) {
```

Comment:
This branch could be moved before `const transform = ...` to save a memory allocation for an unused reference